CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+We follow [Semantic Versioning](http://semver.org/) as a way of measuring stability of an update. This
+means we will never make a backwards-incompatible change within a major version of the project.
+
+## v3.0.0
```

Comment:
Since this appears to be separate from the base project, do we want to have an elevated version number here or take advantage of the opportunity to start over with version numbering?

My thinking is that we can start with pre-1.0 numbering to keep us from being fettered by an API we haven't fully fleshed out yet (per semver guidelines on stability and pre-1.0 versions). On the other hand we might want to keep the version numbers similar across tor_core and tor. Thoughts?